ee/cis/win-10/cis-policy-queries.wip.yml

```diff
@@ -0,0 +1,448 @@
+    'Computer Configuration\Policies\Windows Settings\Security Settings\System Services\Routing and Remote Access'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Services\\RemoteAccess\\Start' AND data == 4);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_5.26
+  contributors: marcosd4h
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'Server (LanmanServer)' is set to 'Disabled'
+  platforms: win10
+  platform: windows
+  description: |
+    The service supports file, print, and named-pipe sharing over the network for this computer.
+  resolution: |
+    Automatic method:
+    Ask your system administrator to establish the recommended configuration via domain GP, set the following UI path to 'Disabled':
+    'Computer Configuration\Policies\Windows Settings\Security Settings\System Services\Server'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Services\\LanmanServer\\Start' AND data == 4);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_5.27
+  contributors: marcosd4h
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'Simple TCP/IP Services (simptcp)' is set to 'Disabled' or to 'Not Installed' 
+  platforms: win10
+  platform: windows
+  description: |
+    The service supports the following TCP/IP services: Character Generator, Daytime, Discard, Echo, and Quote of the Day.
+  resolution: |
+    Automatic method:
+    Ask your system administrator to establish the recommended configuration via domain GP, set the following UI path to 'Disabled' or to 'Not Installed':
+    'Computer Configuration\Policies\Windows Settings\Security Settings\System Services\Simple TCP/IP Services'
+  query: |
+    SELECT CASE 
+        WHEN NOT EXISTS (SELECT * FROM registry WHERE key = 'HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Services\\simptcp\\Start') THEN 1 
+        WHEN (SELECT data FROM registry WHERE key = 'HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Services\\simptcp\\Start') == '4' THEN 1 
+        ELSE 0 
+    END AS result;    
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_5.28
+  contributors: marcosd4h
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'SNMP Service (SNMP)' is set to 'Disabled' or to 'Not Installed'
+  platforms: win10
+  platform: windows
+  description: |
+    The service enables Simple Network Management Protocol (SNMP) requests to be processed by this computer.
+  resolution: |
+    Automatic method:
+    Ask your system administrator to establish the recommended configuration via domain GP, set the following UI path to 'Disabled' or to 'Not Installed':
+    'Computer Configuration\Policies\Windows Settings\Security Settings\System Services\SNMP Service'
+  query: |
+    SELECT CASE 
```

Comment:
When we sync can you explain this sql query for me. I think I have a use case for a similar query on one of my tickets, but wanted to make sure before implementing.

Reply:
sure, the idea of the query is to detect when the registry value does not exist ('Not Installed' state) or when it does exist and it is set to 0 ('Disabled' state). We can discuss this one on a sync